pages/en/network-transition-faq.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Network Transition FAQ
+---
+
+Developers will have plenty of time to migrate their subgraphs to the decentralized network. Exact timelines will vary from chain to chain based on Indexer and network readiness-the hosted service will not end support for all chains at once and will not be sunset abruptly.
+
+Each chain on the hosted service, including Ethereum, will sunset gradually as it is supported on the decentralized network to achieve feature parity and a high quality of service. This will happen on a chain-to-chain basis with help from Indexers in the [MIPs program](https://thegraph.com/blog/mips-multi-chain-indexing-incentivized-program/), to enable full support for each chain on the decentralized network.
+
+To add more clarity around continued support for each chain on the hosted service, these FAQs answer common questions regarding the specifics of the network transition process. If you would like to start the subgraph migration process now, here is a [step-by-step guide](https://thegraph.com/blog/how-to-migrate-ethereum-subgraph). To skip to the migration FAQ, [click here](#migration-fa-qs-answers-to-common-questions).
+
+# Hosted Service Sunset FAQs
```

Comment:
We shouldn't be using `h1`s in markdown content, as there should be a single `h1` per page and it's the post's title (automatically rendered by the layout). So this should be `##` (so the ones below should be `###`, etc.).

Reply:
On it

Reply:
In addition to being bad for SEO (and possibly assistive technology, but not sure about that), the styles are weird because an `h1` doesn't have top margin:

![CleanShot 2022-10-04 at 12 20 39@2x](https://user-images.githubusercontent.com/1059139/193872788-ca997993-0333-4b36-b678-da039b6e840f.png)

(The gap between the previous paragraph and the heading should be larger.)